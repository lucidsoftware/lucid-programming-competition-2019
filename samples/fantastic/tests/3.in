There will be 10 problems