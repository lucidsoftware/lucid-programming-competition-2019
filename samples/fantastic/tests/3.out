There will be 10 problems and it will be fantastic